### DISCUSSION TITLE
Error when running mikro-orm-esm

### DISCUSSION THREAD

**Discussion by eli23456** [mikro-orm/mikro-orm#5147] [General]
### Describe the bug

I am following the instructions to set up for the first time in the docs. When I get to the [Setting up CLI section](https://mikro-orm.io/docs/guide/first-entity#setting-up-cli), I try to run `npx mikro-orm-esm debug` and I see the following error on the command line:
`'"-S"' is not recognized as an internal or external command, operable program or batch file.`

### Reproduction

I can provide steps if needed but I simply followed the steps in the [docs](https://mikro-orm.io/docs/guide/first-entity)

### What driver are you using?

@mikro-orm/sqlite

### MikroORM version

6.0.4

### Node.js version

20.11.0

### Operating system

Windows

### Validations

- [X] Read the [Contributing Guidelines](https://github.com/mikro-orm/mikro-orm/blob/master/CONTRIBUTING.md).
- [X] Read the [docs](https://mikro-orm.io/docs).
- [X] Check that there isn't [already an issue](https://github.com/mikro-orm/mikro-orm/issues) that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/mikro-orm/mikro-orm/discussions) or join our [Slack](https://join.slack.com/t/mikroorm/shared_invite/enQtNTM1ODYzMzM4MDk3LWM4ZDExMjU5ZDhmNjA2MmM3MWMwZmExNjhhNDdiYTMwNWM0MGY5ZTE3ZjkyZTMzOWExNDgyYmMzNDE1NDI5NjA).
- [X] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by eli23456** [comment]
I am able to workaround by using node to run the script
```
node --loader ts-node/esm --no-warnings=Experimental ./node_modules/@mikro-orm/cli/esm.js
```

**Comment by B4nan** [comment]
We already provide windows binary for the CLI here (without the `-S` argument)

https://github.com/mikro-orm/mikro-orm/blob/master/packages/cli/src/esm.cmd

That's the script you should be using, it should be used automatically on windows afaik. But I do have a hunch on why this is not working - the bin script is specifying the `.js` extension [here](https://github.com/mikro-orm/mikro-orm/blob/master/packages/cli/package.json#L50). Can you try to change that to just `"mikro-orm-esm": "./dist/esm"` without the extension? That might work for both linux and windows and pick the right extension automatically. If that won't help, we could add yet another binary for windows (so `"mikro-orm-esm-win": "./dist/esm.cmd"`), although that feels pretty weird.

**Reply by eli23456** [reply]
I upgraded from v6.0.4 to v6.0.5 and it works now!

**Reply by B4nan** [reply]
Great, thanks for the confirmation!
